code/modules/surgery/tools.dm

```diff
@@ -189,7 +187,7 @@
 	hitsound = 'sound/weapons/circsawhit.ogg'
 	throwhitsound =  'sound/weapons/pierce.ogg'
 	flags_1 = CONDUCT_1
-	force = 15
+	force = 8
```

Comment:
15 is not a bad weapon, same damage as a welding tool and 3 more than a toolbox. 8 is way too low imo